src/models/compass.ts

```diff
@@ -0,0 +1,12 @@
+import {SpecQueryModelGroup} from 'compassql/build/src/model';
+
+export interface Compass {
```

Comment:
I see.  Can you be more specific about what you are thinking about?  

(In any case, I feel like Compass is a vague "catch all" name.)